lib/grammers-client/src/types/media.rs

```diff
@@ -49,6 +49,94 @@ pub enum Media {
     Contact(Contact),
 }
 
+pub trait MultiSized {
```

Comment:
I'm okay with improving the way photo sizes but I think this should be in a separate PR (perhaps merged first).

Reply:
no problem. I put it all together because without it most downloads fail

Reply:
https://github.com/Lonami/grammers/pull/146